lib/bundler/cli.rb

```diff
@@ -85,6 +85,9 @@ def help(cli = nil)
       end
     end
 
+    # Ensure `bundle help --no-color` is valid
+    all_commands["help"].disable_class_options = false
```

Comment:
It'd be great to get that fix (and also my io-console PR, if possible), into a release that we could pull in. Thanks so much for your work on this!